src/test/java/org/jenkinsci/plugins/scriptsecurity/sandbox/groovy/SecureGroovyScriptTest.java

```diff
@@ -96,6 +97,16 @@ public class SecureGroovyScriptTest {
 
     @Rule public TemporaryFolder tmpFolderRule = new TemporaryFolder();
 
+    private void addPostBuildAction(HtmlPage page) throws IOException {
+        String displayName = r.jenkins.getExtensionList(BuildStepDescriptor.class).get(TestGroovyRecorder.DescriptorImpl.class).getDisplayName();
+        try {
+            page.getAnchorByText(displayName).click();
+        } catch (ElementNotFoundException enf) {
+            HtmlForm config = page.getFormByName("config");
+            r.getButtonByCaption(config, displayName).click();
+        }
```

Comment:
Same feedback as in https://github.com/jenkinsci/text-finder-plugin/pull/191#pullrequestreview-1596594158 about if/else vs try/catch